tests/python/unittest/test_loss.py

```diff
@@ -292,7 +292,9 @@ def test_huber_loss():
     assert mod.score(data_iter, eval_metric=mx.metric.Loss())[0][1] < 0.05
 
 
-@with_seed(1234)
+# @zachgk: Getting ride of fixed seed as flakiness should not occur with increased tolerance,
+# tracked at https://github.com/apache/incubator-mxnet/issues/11695
```

Comment:
We just decided that this is not needed, please simply link the issue in the description section of your PR